docs/workflow_solutions/using_anaconda.md

```diff
@@ -260,6 +260,126 @@ Now we can be sure that the correct versions of the software will be installed o
     The example above is provided only for illustration purposes. The error has since been fixed, but the example above really happened and is helpful to explain version pinning.
 <!-- markdownlint-enable MD046 -->
 
+#### Update packages in an environment
+
+To ensure packages and their dependencies are all up to date, it is a best practice to regularly update installed packages, and libraries in your activated environment.
+
+```bash
+
+conda update -—all
+
+```
+
+#### Deleting an Environment
+
+To delete an environment, use the following command. Remember to replace `<env>` with the existing environment name.
+
+```bash
+
+conda env remove —-name <env>
+
+```
+
+#### Sharing your environment file
+
+To share your environment for collaboration, there are primarily 3 ways to export environments, the below commands show how to create environment files that can be shared for replication. Remember to replace `<env>` with the existing environment name.
+
+1. Cross-Platform Compatible
+
+    ```bash
+
+    conda env export --from-history > <env>.yml 
+
+    ```
+
+1. Platform + Package Specific
+
+    Create .yml file to share, replace `<envname>` (represents the name of your environment) and `<env>` (represents the name of the file you want to export) with preferred names for file.
+
+    ```bash
+
+    conda env export <envname> > <env>.yml 
+
+    ```
+
+1. Platform + Package + Channel Specific
+
+    ```bash
+
+    conda list —-explicit > <env>.txt
+
+                OR
+
+    conda list —-explicit > <env>.yml
+
+    ```
+
+#### Activating/Using a shared environment file
+
+To use an environment file shared with you, navigate into the folder where you have the .yml file saved. When in the folder, use use the command to create .yml file in your terminal, replace `<env>` with the environment file name.
+    ```bash
+    conda env create -—file <env>.yml
+    ```
+
+#### Switching between environments
+
+Switching between two environments containing different libraries, packages, and dependencies can be carried out following the below steps.
+
+1. Activate an environment using the below command
+
+    ```bash
+
+    conda activate <env>
+
+    ```
+
+To switch the environment, you can do one of the below options using the below commands
+
+1. First deactivate the environment with. Then you activate the second environment with. replace `<env2>` with name of other environment.
+
+    ```bash
+    conda deactivate 
+
+        AND THEN
```

Comment:
Since this is a code block, make this line a comment with the `#` character. Let's leave it at the same level of indentation. I don't think we need the extra lines between each line.

Like this

```bash

conda deactivate
# and then
conda activate env2

```

While we're at it, I want to avoid using `<name>` constructions. I know we've used it in articles in the past, but lets avoid it as much as possible.